package.json

```diff
@@ -5,16 +5,17 @@
     "start": "start-storybook -p 9009",
     "eslint": "eslint ./src",
     "eslint:fix": "eslint --fix ./src",
-    "build": "nwb build-react-component",
-    "build:dist": "nwb build-react-app",
+    "build": "npm run build:lib && npm run build:es",
+    "build:lib": "rimraf lib && babel src -d lib",
+    "build:es": "rimraf es && babel --config-file ./config/babel/es.json src -d es",
+    "build:dist": "webpack -p es/index.js dist/index.js",
```

Comment:
i'm wondering if this command should maybe be: `npm run build && webpack -p ...`

Reply:
yeah that was bothering me a bit too, it would make some builds take a little longer but I agree better to be safe - `npm run build:es && ...` should suffice